source/services/horizon.rst

```diff
@@ -92,3 +92,72 @@ Large Horizon table for django_session
      -rw-rw----  1 42434 42434 1.6K Dec  5 15:02 django_session.frm
      -rw-rw----  1 42434 42434 9.0M Dec  5 15:04 django_session.ibd
      ...
+
+Customize horizon theme
+=======================
+
```

Comment:
Some short text about what a theme is and why one might want to change it might be helpful. Also a link to the upstream docs https://docs.openstack.org/kolla-ansible/latest/reference/shared-services/horizon-guide.html#extending-the-default-local-settings-options